libs/api/domains/cms/src/lib/models/featuredArticles.model.ts

```diff
@@ -0,0 +1,57 @@
+import { Field, ID, ObjectType } from '@nestjs/graphql'
+
+import { IArticle, IFeaturedArticles } from '../generated/contentfulTypes'
+
+import { SystemMetadata } from 'api-cms-domain'
+import { Image, mapImage } from './image.model'
+
+@ObjectType()
+export class FeaturedArticle {
+  @Field(() => ID)
+  id: string
+
+  @Field({ nullable: true })
+  title: string
+
+  @Field()
+  slug: string
+
+  @Field()
+  isApplication: boolean
+}
+
+export const mapFeaturedArticle = ({
+  sys,
+  fields,
+}: IArticle): FeaturedArticle => ({
+  id: sys.id,
+  title: fields.title ?? '',
+  slug: fields.slug ?? '',
+  isApplication: fields.processEntry ? true : false,
+})
```

Comment:
This model might not be needed since we can get this data via the `Article` model.
Using the `Article` model might also be preferable to ensure you have access to all article fields in articles under `FeaturedArticles`

Reply:
That's what we tried first but it won't work as the depth limit in the Contentful client is set to 4. I'm not sure if it's a good idea to increase it.

Reply:
I see, you can bypass this issue by using nested resolvers.
An example of this can be found in the `relatedArticles` field of the `Article` model, with the resolver being in `cms.resolver.ts`.

Reply:
Turns out that using `Article` works fine after all. I have simplified the model by using the `Article` model instead.